troubleshooting/troubleshooting-ios-cannot-load-iframe.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Cannot Load iframe on iOS Devices When Using Cordova 5.0.0 or Later
+description: iframe does not open on iOS devices with Cordova 5.0.0 and 5.2.0.
+type: troubleshooting
+page_title: Cordova for iOS 4.0.1 or Later Apps do not Load iframe
+slug: ios-cannot-load-iframe
+position: 
+tags: appbuilder, ios, cordova 
+teampulseid: 
+ticketid: 1060248
+pitsid: 
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code (AppBuilder)</td>	
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid (Cordova app)</td>	
+  </tr>
+
+  <tr>
+    <td>Mobile OS</td>
+    <td>iOS</td>	
+  </tr>
+  <tr>
+    <td>Cordova framework version</td>
+    <td>Cordova 5.0.0 and later</td>	
+  </tr>
+</table>
+
+## Description
+
+Hybrid applications that are deployed on an iOS device do not show the content of an iframe.
+
+## Steps to Reproduce
+
+1. [Create](http://docs.telerik.com/platform/appbuilder/cordova/creating-your-app/creating-a-new-app) or open a Cordova application.
+1. [Change the target Cordova version](http://docs.telerik.com/platform/appbuilder/cordova/configuring-your-app/configure-target-framework) to 5.0.0 or later.
+1. Include an `<iframe src=""></iframe>` tag to an HTML view.
+1. Deploy the app to an iOS device.
+
+## Cause
+
+Cordova 5.0.0(Cordova for iOS 4.0.1) incorporates the Cordova whitelisting system, which provides a configurable security policy. As Cordova 5.0.0 is marked experimental in AppBuilder, the project template does not include the required configuration to enable whitelisting.
+
+## Solution
+
+To enable whitelisting in your iOS application, complete the following steps:
+
+1. Open the **iOS Config.xml** file in your project, as described [here](http://docs.telerik.com/platform/appbuilder/cordova/configuring-your-app/edit-configuration).
+1. In that file, add the following code before the closing `</widget>` tag: 
+
+ 	```XML
+	<feature name="IntentAndNavigationFilter">
+		<param name="ios-package" value="CDVIntentAndNavigationFilter"/>
+		<param name="onload" value="true"/>
+	</feature>
+	```
+1. Whitelist the links of the iframes.<br/>You can add multiple  `<allow-navigation>` elements.
+
+	```XML
+	<allow-navigation href="http://example.com/" />
+	```
+
+## Notes
+
+For testing purposes, you can whitelist all URLs at once.
+
+```XML
+<allow-navigation href="*" />
+```
+Before moving your application to production, you should formulate a whitelist and allow access to specific network domains and subdomains as described above.
+
+## See Also
+* [Cordova Whitelist Guide](https://cordova.apache.org/docs/en/latest/guide/appdev/whitelist/)
+* [Cordova Whitelist Plugin](https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-whitelist/)
+* [iframe not loading after upgrade to Cordova iOS 4.1.1](http://stackoverflow.com/questions/37067342/iframe-not-loading-after-upgrade-to-cordova-ios-4-1-1)
```

Comment:
[iframe not loading after upgrade to Cordova iOS 4.1.1 on Stackoverflow](http://stackoverflow.com/questions/37067342/iframe-not-loading-after-upgrade-to-cordova-ios-4-1-1)